compat/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#   This file is part of the dbus-cxx library.                            
```

Comment:
This compat layer works for now, although longer-term I need to take the time to understand what methods are actually needed and if we can replace them with native Windows functions.

Reply:
Sure, perhaps a layer of abstraction with posix/win implementations would be a cleaner approach.